src/pages/docs/contributing.md

```diff
@@ -1,13 +1,70 @@
 ---
-title: Contributing
+title: Contributing to Protocol
 order: 2
 ---
 
-We've done our best to make the barrier of entry to contributing code
-to Protocol as low as possible. You can even do a lot of stuff directly
-through GitHub without actually installing Protocol locally, but if you
-want to do any complex coding work you'll eventually need to set up a
-local development environment.
+Protocol, like everything else Mozilla makes, is open source. We welcome
```

Comment:
It might be helpful to link to our [main/root open source/contribution guide on MDN](https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Introduction). _(Side question: Is this actually the root contribution guide for Mozilla?)_

```suggestion
Protocol, like everything else Mozilla makes, is [open source](https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Introduction). We welcome
```